doc/introduction/circuits.rst

```diff
@@ -180,13 +180,13 @@ In the quantum function you can now use your own labels to address wires:
 
 Allowed wire labels can be of the following types:
 
-* *strings* like ``wires=['a', 'd', 'b', ...]`` or ``wires=['ancilla', 'q1', 'q2', ...]``,
+* *strings* like ``wires=['a', 'd', 'b', ...]`` or ``wires=['auxiliary', 'q1', 'q2', ...]``,
```

Comment:
I recall that auxiliary would be preferred to ancilla in terms of terminology, but can't recall the exact reason. What was it?

Reply:
Ancilla basically means a particular type of female slave in the Roman empire.

Reply:
Thanks! :+1: